bin/po2android.js

```diff
@@ -1,12 +1,19 @@
 #!/usr/bin/env node
 
 const gettextParser = require( 'gettext-parser' ),
-	fs = require( 'fs' );
+	fs = require( 'fs' ),
+	crypto = require( 'crypto' );
 
 const indent = '    ';
 
+/**
+ * Encode a raw string into an Android-compatible value
```

Comment:
Should we also encode quotes and double-quotes?

Reply:
Yep, `JSON.stringify` is going to take care of that

Reply:
Single quotes don't need to be encoded though as they'll be contained in a double quote string